src/csharp/Microsoft.Spark/ML/Feature/StopWordsRemover.cs

```diff
@@ -0,0 +1,173 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using Microsoft.Spark.Interop;
+using Microsoft.Spark.Interop.Ipc;
+using Microsoft.Spark.Sql;
+using Microsoft.Spark.Sql.Types;
+
+namespace Microsoft.Spark.ML.Feature
+{
+    /// <summary>
+    /// A <see cref="StopWordsRemover"/> feature transformer that filters out stop words from input.
+    /// </summary>
+    public class StopWordsRemover : FeatureBase<StopWordsRemover>, IJvmObjectReferenceProvider
+    {
+        private static readonly string s_stopWordsRemoverClassName =
+            "org.apache.spark.ml.feature.StopWordsRemover";
+
+        /// <summary>
+        /// Create a <see cref="StopWordsRemover"/> without any parameters
+        /// </summary>
+        public StopWordsRemover() : base(s_stopWordsRemoverClassName)
+        {
+        }
+
+        /// <summary>
+        /// Create a <see cref="StopWordsRemover"/> with a UID that is used to give the
```

Comment:
The summary tags should end with a full stop (.), the `public StopWordsRemover(string uid)` constructor is missing it